### PR TITLE
LPS-143885 Rename random keys

### PR DESCRIPTION
**Pull request by LuismiBarcos**
This PR is just a follow-up for [this](https://github.com/brianchandotcom/liferay-portal/pull/112055#issuecomment-1011338345).

In the FDS tags, we have the property `randomNamespace` that should be unique and follow a common naming pattern. With this PR I'm changing this to follow this pattern and be unique

cc/ @markocikos @javierdearcos

### CODE REVIEW

**Comment by LuismiBarcos** [comment]
ci:forward

File: modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/resources/META-INF/resources/classic_display/init.jsp [hunk 1]
```diff
@@ -25,7 +25,7 @@ page import="com.liferay.portal.kernel.util.PortalUtil" %>
 <liferay-theme:defineObjects />
 
 <%
-String randomNamespace = PortalUtil.generateRandomKey(request, "taglib_step_tracker") + StringPool.UNDERLINE;
+String randomNamespace = PortalUtil.generateRandomKey(request, "tablig_frontend_data_set_classic_display") + StringPool.UNDERLINE;
```
Comment (markocikos) [inline modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/resources/META-INF/resources/classic_display/init.jsp:28]: Typo? Probably it should be taglib and not tablig :slightly_smiling_face:

**Review by markocikos — COMMENTED** [review]
Minor typo inline, otherwise looks good!

File: modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/resources/META-INF/resources/classic_display/init.jsp (hunk 1 above)
Comment (LuismiBarcos) [reply modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/resources/META-INF/resources/classic_display/init.jsp:28]: 😱

**Comment by LuismiBarcos** [comment]
ci:stop

**Comment by LuismiBarcos** [comment]
Thanks for the advice @markocikos 😊 😊 😊

**Comment by LuismiBarcos** [comment]
ci:forward

**Pull request closed without merging**